sdk/communication/AzureCommunicationChat/Package.swift

```diff
@@ -1,39 +0,0 @@
-// swift-tools-version:5.1
-//  The swift-tools-version declares the minimum version of Swift required to build this package.
-//
-// --------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator.
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
-// --------------------------------------------------------------------------
-
-import PackageDescription
```

Comment:
This file shouldn't be deleted. Autorest will regenerate it and we have certain CI checks that may use it in the future.

Reply:
My bad, I thought this was remnant from preview2, reverted.